src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -478,6 +418,70 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
         glyphToCompletionKind <- glyphToCompletionKindGenerator clientCapabilities
         glyphToSymbolKind <- glyphToSymbolKindGenerator clientCapabilities
 
+        let tryGetParseResultsForFile fileName pos = asyncResult {
```

Comment:
The next few lines are basically dependency injection for our codefixes.  This should theoretically make it easier to test the fixes, but it also seems to make them easier to write/work with IMO.